besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2595,17 +2583,6 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
       builder.setGenesisConfig(genesisConfig());
 
       if (networkId == null) {
-        // if no network id option is defined on the CLI we have to set a default value
-        // from the
-        // genesis file.
-        // We do the genesis parsing only in this case as we already have network id
-        // constants
-        // for known networks to speed up the process.
-        // Also we have to parse the genesis as we don't already have a parsed version
-        // at this
-        // stage.
-        // If no chain id is found in the genesis as it's an optional, we use mainnet
-        // network id.
```

Comment:
If no chain id is found in the genesis, use mainnet network id.

Reply:
^ maybe a short comment is still useful here?

Reply:
53bde8a